### PR TITLE
feat: [DEL-6035]: show delegate logs in ui

### PR DESCRIPTION
**Pull request by abhinavrastogi-harness**
#### Screenshots

image

Screenshot 2023-02-14 at 1 03 32 PM
Screenshot 2023-02-14 at 12 54 52 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: configs/restful-react.config.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const customGenerator = require('../scripts/swagger-custom-generator.js')
 module.exports = {
   portal: {
     output: 'src/services/portal/index.tsx',
-    // url: 'https://localhost:9090/api/swagger.json',
```
Comment (vikrant-harness) [inline configs/restful-react.config.js:17]: is this intentional ?

Comment (abhinavrastogi-harness) [reply configs/restful-react.config.js]: no.. fixing

**Comment by abhinavrastogi-harness** [comment]
retrigger titlecheck

**Review by satyam-harness — APPROVED** [review]

**Review by avinashmadhwani02 — DISMISSED** [review]

**Comment by abhinavrastogi-harness** [comment]
retrigger titlecheck

**Review by sunnykesh-harness — DISMISSED** [review]

**Review by kapilkumawat86 — DISMISSED** [review]

**Review by vkbansal-harness — APPROVED** [review]

**Review by avinashmadhwani02 — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**